### PR TITLE
RUN-2483 MPI CRD validation

### PR DESCRIPTION
**Pull request by gshaibi**

### CODE REVIEW

File: cmd/job/submit/submit_mpi.go [hunk 1]
```diff
@@ -162,8 +165,28 @@ func (submitArgs submitMPIJobArgs) check() error {
 	return nil
 }
 
+func isMpiJobCrdExist() (bool, error) {
+	crds, err := kubectl.GetCRDs()
```
Comment (omer-dayan) [inline cmd/job/submit/submit_mpi.go:169]: Why run 'kubectl get crds' and then check in the list and not run 'kubectl get crd mpijobs'?

File: pkg/util/kubectl/kubectl.go [hunk 2]
```diff
@@ -413,6 +413,11 @@ func PortForward(ports []string, PortForwardAddr string, serviceName string, nam
 	return kubectlAttched(args)
 }
 
+func GetCRDs() (string, error) {
+	args := []string{"get", "customresourcedefinitions.apiextensions.k8s.io"}
+	return kubectl(args)
```
Comment (omer-dayan) [inline pkg/util/kubectl/kubectl.go:418]: We much prefer use client-go instead of direct kubectl commands

Comment (gshaibi) [reply pkg/util/kubectl/kubectl.go:418]: Couldn't find API for CRDs

File: cmd/job/submit/submit_mpi.go (hunk 1 above)
Comment (gshaibi) [reply cmd/job/submit/submit_mpi.go:169]: Wanted to generalize it.
I'll change it.

File: pkg/util/kubectl/kubectl.go (hunk 2 above)
Comment (omer-dayan) [reply pkg/util/kubectl/kubectl.go]: You can use unstructured dynamic client

File: pkg/util/kubectl/kubectl.go [hunk 3]
```diff
@@ -413,6 +413,21 @@ func PortForward(ports []string, PortForwardAddr string, serviceName string, nam
 	return kubectlAttched(args)
 }
 
+func IsCrdExist(crdName string) (bool, error) {
```
Comment (omer-dayan) [inline pkg/util/kubectl/kubectl.go:416]: No need

**Review by omer-dayan — APPROVED** [review]

**Pull request merged**